datasets/ebirdst.yaml

```diff
@@ -0,0 +1,43 @@
+Name: eBird Status and Trends Model Results
+Description: |
+  The eBird Status and Trends project generates estimates of bird 
+  occurrence and abundance at a high spatiotemporal resolution. 
+  This dataset represents the primary modeled results from the
+  analysis workflow and are designed for further analysis, 
+  synthesis, visualization, and exploration.
+Documentation: https://ebird.org/science/status-and-trends
+Contact: https://help.ebird.org/customer/en/portal/emails/new
+ManagedBy: Cornell Lab of Ornithology
+UpdateFrequency: Yearly
+Tags:
+  - biodiversity
```

Comment:
Please add `aws-pds` as a tag.

Reply:
Added in 7d5c6ae3ac0257bfa0f33bbe5fdc10475162f9ef